pages/lists/[listId]/activity.tsx

```diff
@@ -61,21 +71,54 @@ export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
       activityData: {
         contributorStats: mostActiveData,
         topContributor: mostActiveData?.data?.length ? mostActiveData.data[0] : null,
+        projectData: projectData ?? [],
       },
     },
   };
 };
 
 const ListActivityPage = ({ list, numberOfContributors, isError, activityData }: ContributorListPageProps) => {
+  const [range, setRange] = useState(30);
   const isOwner = false;
+  const [level, setLevel] = useState(0);
   const {
     data: contributorStats,
     isLoading,
     isError: isMostActiveError,
-    setRange,
     setContributorType,
     contributorType,
-  } = useMostActiveContributors({ listId: list!.id, initData: activityData.contributorStats.data });
+  } = useMostActiveContributors({ listId: list!.id, initData: activityData.contributorStats.data, range });
+
+  const { setRepoId, error, data: projectContributionsByUser } = useContributorsByProject(list!.id, range);
+
+  const { data: projectData, error: projectDataError } = useContributionsByProject({
+    listId: list!.id,
+    range,
+    initialData: activityData.projectData,
+  });
+
+  const onHandleClick = ({ id }: { id: string }) => {
+    const repoId = Number(id.split(":")[1]);
+    setRepoId(repoId);
+    setLevel(level + 1);
+  };
+  const treemapData = {
+    id: "root",
+    children:
+      level === 0
+        ? (projectData ?? []).map(({ org_id, project_id, repo_id, contributions }) => {
+            return {
+              id: `${org_id}/${project_id}:${repo_id}`,
```

Comment:
Can we take out the repo_id? It shows when you hover over the repo

```suggestion
              id: `${org_id}/${project_id}`,
```

Reply:
I need it to drill down to the contributors. Let me see why it's not being split on the tooltip. It's fine for the regular title.

Reply:
Done. See [`d3f3bb3` (#1853)](https://github.com/open-sauced/app/pull/1853/commits/d3f3bb378300046fc524f7e7d51305693dfd4a0f)